errors.go

```diff
@@ -0,0 +1,11 @@
+package moira
+
+//SenderBrokenContactError means than sender has no way to send message to contact.
+// Maybe receive contact was deleted, blocked or archived.
+type SenderBrokenContactError struct {
+	SenderError error
+}
+
+func (e *SenderBrokenContactError) Error() string {
```

Comment:
Let's use concrete receiver instead of pointer

Reply:
fixed